chaos-monkey-spring-boot/src/test/java/de/codecentric/spring/boot/chaos/monkey/endpoints/ChaosMonkeyRequestScopeRestEndpointIntegration.java

```diff
@@ -273,7 +275,7 @@ void postToEnableChaosMonkey() {
         testRestTemplate.postForEntity(baseUrl + "/enable", null, String.class);
 
     assertEquals(HttpStatus.OK, result.getStatusCode());
-    assertEquals("Chaos Monkey is enabled", result.getBody());
+    assertTrue(Objects.requireNonNull(result.getBody()).startsWith("Chaos Monkey is enabled"));
```

Comment:
As soon as we introduced an object (see comment above). You could make assertions again. 
it would be great if we make a basic assertion that the correct time would be returned.

We could for example remember the time before we send the request and make an assertion that the time is returned should be later than the remembered time.

Reply:
So it would still be assertTrue but just checking that the time stored in object returned is later than time recorded before assertion statement?

Reply:
Yeah it could be two asserts. One which asserts the text/status "Chaos Monkey is enabled" and one the time yes